main.go

```diff
@@ -1,29 +1,46 @@
+// Package main is responsible for handling CLI entrypoint of modfmt
 package main
 
 import (
 	"fmt"
 	"log"
 	"os"
 	"slices"
+
+	"github.com/PaddleHQ/modfmt/pkg/modfmt"
```

Comment:
I moved this to a `pkg` directory.
Having it in a separate package makes the code cleaner.
Having it in `pkg` means that it can be imported into other tools